### PR TITLE
Privacy policy updates. Closes #2276

### LINKED ISSUE #2276: Clarify privacy policy
While re-reading the PP we noticed it's a little unclear about how raw IPs get logged. We have them in our raw load balancer logs and in a couple of other places; all of which are deleted after a short amount of time (cloudfront is 28 days; other places keep up to 90 days). We should make sure this is communicated clearly.

### PR DESCRIPTION
**Pull request by mattgodbolt**

### CODE REVIEW

**Review by RubenRBS — APPROVED** [review]
Let me know if you like the changes :)

File: Makefile [hunk 1]
```diff
@@ -112,6 +112,6 @@ changelog:  ## Create the changelog
 	python ./etc/scripts/changelog.py
```
Comment (mattgodbolt) [inline Makefile:112]: can you update this one while you're at it?

File: etc/scripts/politic.py [hunk 2]
```diff
@@ -26,25 +26,26 @@
...
+    policy_last_commit = subprocess.check_output(['git', 'log', '-1', '--format=%H', policy_path]).decode('utf-8').rstrip()
+    print(f'Setting policy {policy_path} last updated time to {policy_last_time} with commit {policy_last_commit}')
     f = open(policy_path, 'r')
     file_lines = f.readlines()
     f.close()
     with open(policy_path, 'w') as f:
         for line in file_lines:
             if re.match(date_placeholder_regex, line):
-                f.write(re.sub(date_placeholder_regex, '\\1Last changed on: {}\\2'.format(privacy_output), line))
+                f.write(re.sub(date_placeholder_regex, f'\\1Last changed on: <time id="changed-date" datetime="{policy_last_time}">{policy_last_time}</time> <i>(<a href="https://github.com/compiler-explorer/compiler-explorer/commit/{policy_last_commit}" target="_blank">diff</a>)</i>\\2', line))
```
Comment (mattgodbolt) [inline etc/scripts/politic.py:48]: oh neat!

**Review by mattgodbolt — COMMENTED** [review]
looks good! (can't approve as this is my change)

**Pull request merged**